src/instance/minecraft/CommandsManager.ts

```diff
@@ -76,6 +75,20 @@ export class CommandsManager extends EventHandler<MinecraftInstance> {
       return true
     }
 
+    if (commandName === 'help' || commandName === 'command') {
+      if (arguments_.length <= 0) {
+        await minecraftInstance.send(`/gc Commands: ${this.commands.map((command) => command.name).join(', ')}`)
+        return false
+      }
+      const command = this.commands.find((c) => c.triggers.includes(arguments_[0]))
+      if (command == undefined) return false
```

Comment:
Add `command not found (try ${prefix}help)` message or something along these lines. then return `true` since the command has been processed.